src/applications/vaos/sass/vaos.scss

```diff
@@ -232,3 +232,12 @@
 h1:focus {
   outline: none !important;
 }
+
+.vaos-form__checkboxgroup {
```

Comment:
I shouldn't need to do this, but checkbox groups seem messed up across our forms right now.